keylime/ima/file_signatures.py

```diff
@@ -125,7 +130,7 @@ def _get_keyidv2(pubkey):
         keydigest = digest.finalize()
         return int.from_bytes(keydigest[16:], "big")
 
-    def add_pubkey(self, pubkey, keyidv2):
+    def add_pubkey(self, pubkey: SupportedKeyTypes, keyidv2: Union[int, None]) -> None:
```

Comment:
Instead of `Union[int, None]` you can use `Optional[int]`. This can be changed in a few places.

Reply:
Done.